### ISSUE TITLE
Importing from NetDeck adds 120 copies of a "NOOOOOOOOOO" card

### ISSUE THREAD

**Issue by emzero** [HearthSim/Hearthstone-Deck-Tracker#2386]
Sorry if this is not an issue with the tracker but with NetDeck, but since it started happening with the latest release (0.14.4) I decided to post it here.

Basically, when I click NetDeck button from Chrome it imports the deck from the web site but it adds 120 copies of a "NOOOOOOOOO" card as shown below:

![screen](https://i.imgur.com/zZUequP.png)

**Comment by livejamie** [comment]
+1 So much clicking

**Comment by Epix37** [comment]
This looks like an issue with NetDeck. @n4ru

"NOOO..." is just a unknown (not found) card.

**Comment by n4ru** [comment]
Cheers thanks for the tag.

This looks particular annoying, I'll get it fixed asap.

Could you give me some examples of decklist sites and exact links that cause this behavior?

**Comment by livejamie** [comment]
I've just been using Hearthpwn I didn't think to try other sites

I assume it's something to do with the new cards

On Wed, Apr 27, 2016 at 6:01 AM, n4ru <notifications@github.com> wrote:

> Cheers thanks for the tag.
>
> This looks particular annoying, I'll get it fixed asap.
>
> Could you give me some examples of decklist sites and exact links that
> cause this behavior?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/HearthSim/Hearthstone-Deck-Tracker/issues/2386#issuecomment-215074313>
>



-- 
Work: http://jamiemart.in
Play: http://livejamie.com
Twit: http://twitter.com/livejamie

**Comment by Endoplasm** [comment]
I'm also having this issue between HearthPwn, NetDeck and HDT. I tried importing decks without any of the new Old Gods cards and still have the same issue. If I make a deck myself on HearthPwn and stay on the deckbuilding page and use NetDeck to import, it works as normal without the 130 "NOOOOO" cards. Hope that helps, good luck.

**Comment by n4ru** [comment]
It's because the deckbuilding page uses a different function to import. I'll be taking a look into this tonight and hopefully fixing the issue assuming it's my end (99% sure it is). Expect an update tomorrow night the latest.

**Comment by Endoplasm** [comment]
Awesome, thanks for the quick response!

**Comment by n4ru** [comment]
Has anyone experienced this issue with other sites? I see the original post mentioned sites but nothing specific. I'm not home so can't check for a few hours.

**Comment by emzero** [comment]
@n4ru I've just checked with both http://www.hearthstonetopdeck.com/ and http://www.hearthstonetopdecks.com/ and it doesn't happen. I guess it's only in HearthPwn.

**Comment by n4ru** [comment]
Okay so I'm almost certain about what's causing the issue then. Should be fixed tonight then.

**Comment by Epix37** [comment]
@n4ru status? Still getting reports for this.

**Comment by Emidomenge** [comment]
I tried 9 hours ago, still not working.

**Comment by Epix37** [comment]
FYI: you can still use HDTs default importing via `IMPORT > FROM WEB`.

**Comment by n4ru** [comment]
I've had such a busy day last two days, I am taking care of this right now!!!

Will be fixed within the hour!

**Comment by n4ru** [comment]
Ok the fix is pushed. This one actually requires an update. Give it another hour to roll out since the chrome store takes a bit.

Thank you for bringing it to my attention everyone!

**Comment by lunarwolfspirit** [comment]
I paid for the chrome extension today (never noticed it became a paid tool) - but am not getting the updates

**Comment by livejamie** [comment]
I'm also getting the same error.

On Sat, Apr 30, 2016 at 7:23 AM, lunarwolfspirit <notifications@github.com>
wrote:

> I paid for the chrome extension today (never noticed it became a paid
> tool) - but am not getting the updates
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/HearthSim/Hearthstone-Deck-Tracker/issues/2386#issuecomment-215968854>
>



-- 
Work: http://jamiemart.in
Play: http://livejamie.com
Twit: http://twitter.com/livejamie

**Comment by n4ru** [comment]
I will recheck this again.

Please make sure you are updated to** >>3.0.11<<** and report back if you are.

**Comment by n4ru** [comment]
I've checked with some others reporting the issue is still around and they all seem to be on 3.0.10. Double check, sometimes Chrome extensions either take a while to fully roll out or do not automatically update. I am almost positive it is fixed (tested personally across a few machines) in 3.0.11.

If this turns out to not be the case, it will be fixed.

**tl;dr Update to 3.0.11.**

**Comment by lunarwolfspirit** [comment]
Yep checked yesterday and the extension was updated on chrome and works perfectly within HDT now

Warm regards,
Alexander Zhao
